crypto/evp/p_rsa_asn1.c

```diff
@@ -108,6 +108,19 @@ static int rsa_pub_decode(EVP_PKEY *out, CBS *params, CBS *key) {
   return 1;
 }
 
+static int rsa_pss_pub_decode(EVP_PKEY *out, CBS *params, CBS *key) {
+  // TODO(shang): add pss paramters decode.
+  RSA *rsa = RSA_parse_public_key(key);
```

Comment:
`RSA_parse_public_key` is outside the current fipsmodule scope. Does that create any problems in the future?

Reply:
This is also used by EVP_PKEY_RSA EVP_PKEY_ASN1_METHOD.

Reply:
Is an issue needed to track `RSA_parse_public_key` outside fipsmodule scope? I tried to create one and don't know how to describe the concern here. Would you provide more details?

Reply:
Chatting offline.